The Economic Times.js

```diff
@@ -0,0 +1,192 @@
+{
+	"translatorID": "1a9a7ecf-01e9-4d5d-aa19-a7aa4010da83",
+	"label": "The Economic Times",
+	"creator": "Sonali Gupta",
+	"target": "http://economictimes.indiatimes.com",
```

Comment:
This is a regular expression and we can therefore be more strict about the beginning, but already now allow possible `https` urls, and escape the points, i.e.
```diff
-"target": "http://economictimes.indiatimes.com",
+"target": "^https?://economictimes\\.indiatimes\\.com",
```
Note: In Scaffold you only need to escape them once, i.e. `\.`.